### PR TITLE
[5.2] Fix double closing Curly braces in inline style

### PR DESCRIPTION
**Pull request by cyrez**
Fix double closing Curly braces in inline style for `:root[data-color-scheme="dark"] {`

### Summary of Changes
Remove the extra curly brace.

### Testing Instructions
Just a look at code.

### Actual result BEFORE applying this Pull Request
Nothing changed

### Expected result AFTER applying this Pull Request
Nothing changed

### Link to documentations
Please select:
- [ ] Documentation link for docs.joomla.org: <link>
- [x] No documentation changes for docs.joomla.org needed

- [ ] Pull Request link for manual.joomla.org: <link>
- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

File: administrator/templates/atum/index.php [hunk 1]
```diff
@@ -97,7 +97,7 @@ function adjustColorLightness($r, $g, $b, $percent)
 		--link-color-rgb: ' . $rd . ',' . $gd . ',' . $bd . ';
         --link-color-rgb-hvr: ' . $linkColorDarkHvr . ';
```
Comment (richard67) [inline administrator/templates/atum/index.php]:
```suggestion
		--link-color-rgb-hvr: ' . $linkColorDarkHvr . ';
```
Could we also replace the 8 spaces here by 2 tabulators so the line is the same as the other above and below?

**Comment by richard67** [comment]
I have tested this item :white_check_mark: successfully on 11e584132a6fb4d402088f639518d9a06f03f114<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44532">issues.joomla.org/tracker/joomla-cms/44532</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/44532">issues.joomla.org/tracker/joomla-cms/44532</a>.</sub>

**Comment by Hackwar** [comment]
Thank you for this fix!

**Comment by cyrezdev** [comment]
Thanks @Hackwar for fixing indentation in the same time 👍

**Pull request merged**